core/src/mindustry/world/blocks/defense/turrets/Turret.java

```diff
@@ -432,7 +432,7 @@ protected void effects(){
 
             fshootEffect.at(x + tr.x, y + tr.y, rotation);
             fsmokeEffect.at(x + tr.x, y + tr.y, rotation);
-            shootSound.at(tile, Mathf.random(0.9f, 1.1f));
+            shootSound.at(x + tr.x, y + tr.y, 1);
```

Comment:
The pitch for the *charge* sound shouldn't be random, but the normal sound should be.